content/200-concepts/100-components/02-prisma-client/240-metrics.mdx

```diff
@@ -379,6 +379,16 @@ prisma_datasource_queries_duration_histogram_ms_sum 1.8407059999999997
 prisma_datasource_queries_duration_histogram_ms_count 5
 ```
 
+
+Metrics of type `histogram` expose three different types of values in the Prometheus format:
```

Comment:
to avoid the repetition of `type` with different meaning:
```suggestion
Metrics of type `histogram` expose three different class of values in the Prometheus format:
```